### PR TITLE
docs: broken links in readme file

### LINKED ISSUE #873: [Docs Bug 🐞 report]: Links in main readme files are dead
### Describe the bug you found in AsyncAPI Docs.

There are multiple links dead in https://github.com/asyncapi/generator#cli-installation

### Attach any resources that can help us understand the issue.

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by sambhavgupta0705**
**Description**

-Updated the broken links in readme file

**Related issue(s)**

Resolves #873

### CODE REVIEW

**Comment by sambhavgupta0705** [comment]
cc @derberg @Florence-Njeri @jonaslagoni

File: README.md [hunk 1]
```diff
@@ -64,15 +64,15 @@ Install both packages using [official installer](https://nodejs.org/en/download/
 ## Using from the command-line interface (CLI)
 
 ### CLI installation
-Learn to install the AsyncAPI CLI from the [installation guide](installation-guide.md).
+Learn to install the AsyncAPI CLI from the [installation guide](https://github.com/asyncapi/generator/blob/master/docs/installation-guide.md).
```
Comment (Florence-Njeri) [inline README.md:67]:
```suggestion
Learn to install the AsyncAPI CLI from the [installation guide](docs/installation-guide.md).
```

File: README.md [hunk 2]
```diff
@@ -64,15 +64,15 @@ Install both packages using [official installer](https://nodejs.org/en/download/
...
 
 ### CLI usage 
 
-Learn more about different ways of using the CLI from the [usage document](usage.md).
+Learn more about different ways of using the CLI from the [usage document](https://github.com/asyncapi/generator/blob/master/docs/usage.md).
```
Comment (Florence-Njeri) [inline README.md:71]:
```suggestion
Learn more about different ways of using the CLI from the [usage document](docs/usage.md).
```

File: README.md [hunk 3]
```diff
@@ -64,15 +64,15 @@ Install both packages using [official installer](https://nodejs.org/en/download/
...
 
 ## Generator version vs Template version
 
-Learn more about versioning from the [versioning document](versioning.md).
+Learn more about versioning from the [versioning document](https://github.com/asyncapi/generator/blob/master/docs/versioning.md).
```
Comment (Florence-Njeri) [inline README.md:75]:
```suggestion
Learn more about versioning from the [versioning document](docs/versioning.md).
```

**Review by Florence-Njeri — CHANGES_REQUESTED** [review]

**Comment by Florence-Njeri** [comment]
> cc @derberg @Florence-Njeri @jonaslagoni

Hey, @sambhavgupta0705 good job fixing the broken links. I have left a few suggestions on the fixes

**Comment by sambhavgupta0705** [comment]
Hi @Florence-Njeri made the suggested changes

**Comment by pratik2315** [comment]
Hey @sambhavgupta0705 i dont see any broken links in the readme🤔

**Comment by sambhavgupta0705** [comment]
@pratik2315 there were 3 links in which  `docs/` prefix was missing

**Comment by pratik2315** [comment]
> @pratik2315 there were 3 links in which `docs/` prefix was missing

Oh okay👍

**Comment by derberg** [comment]
@sambhavgupta0705 when you followup on a review and you are ready for another round from initial reviewer, then please always remember to use the circle-arrow icon 👇🏼 

Screenshot 2022-12-15 at 13 45 58

**Review by Florence-Njeri — APPROVED** [review]

**Comment by sambhavgupta0705** [comment]
> @sambhavgupta0705 when you followup on a review and you are ready for another round from initial reviewer, then please always remember to use the circle-arrow icon 👇🏼
> 
> Screenshot 2022-12-15 at 13 45 58

ohk ,will take care of it from now

**Review by derberg — APPROVED** [review]

**Comment by derberg** [comment]
@sambhavgupta0705 I updated PR title to `docs: `. It is important to remember to use conventional commits properly. If we would merge this PR with `fix:` prefix, or `feat:`, that would trigger release pipeline and patch release for generator would go out, but because of links fix it is not needed.

@Florence-Njeri wanna merge? you know the comment that will trigger that? just add it once all the check are green and the status of PR shows green

**Comment by sambhavgupta0705** [comment]
> @sambhavgupta0705 I updated PR title to `docs: `. It is important to remember to use conventional commits properly. If we would merge this PR with `fix:` prefix, or `feat:`, that would trigger release pipeline and patch release for generator would go out, but because of links fix it is not needed.
> 
> @Florence-Njeri wanna merge? you know the comment that will trigger that? just add it once all the check are green and the status of PR shows green

Okay @derberg ,will take care of this

**Pull request merged**